### DISCUSSION TITLE
Question 41: Can we get this implemented in EMu or in NetX [NC-Library]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1028] [Q&A]
Description: see the small red dot in the lower left of the thumbnail? I often work with many images and it would be very useful to know which images are restricted.  The red “frowny” face appears in each individual record, but changing back and forth one at a time is time consuming.  I blurred the images just as an example

**Comment by magpiedin** [comment]
@rondlg, Nina C & any others -- https://emu.ideas.aha.io/ideas/EMU-I-125

**Reply by rondlg** [reply]
Response: Not currently - this would be a development request to Axiell. There is an existing thread with them requesting that this be available in list view.
Add to the ideas portal if it’s not already there and share back with emu-users so they can upvote.
